### PR TITLE
careers.en.md: page-update

### PR DESCRIPTION
**Pull request by Stanislaw-bnk**

### CODE REVIEW

**Comment by artur-rs** [comment]
@Stanislaw-bnk we have also PL version of the careers site, do you plan to update it as well?

File: content/careers.en.md [hunk 1]
```diff
@@ -24,119 +24,70 @@ careers:
...
+    To make it easier for you to join our R&D team, here are some tips for success: 
+    
+    * Share the results of your work with us and show us your code. If you have already contributed to open-source projects, be sure to tell us about them during the recruitment interview. We'd love to see the results of your work, so feel free to send or show us your repository with your CV.
 
-    Over the years, the community recognized us as coreboot licensed service providers, UEFI Adopters, LVFS Consultants, and Yocto Participants and accepted our membership in OpenPOWER Foundation.
+    * Show us what tools and programming languages you know. Check out our project details and source code in our repositories:
+    [https://github.com/3mdeb](https://github.com/3mdeb), [https://gitlab.com/3mdeb](https://gitlab.com/3mdeb), [https://github.com/Dasharo](https://github.com/Dasharo) and to see if they align with your skills and interests. We value candidates with experience in at least one of the programming languages we work with every day that can immediately support us in daily projects.
```
Comment (artur-rs) [inline content/careers.en.md:36]: @Stanislaw-bnk please remove gitlab 3mdeb link, there is no access without direct membership

**Comment by Stanislaw-bnk** [comment]
> @Stanislaw-bnk we have also PL version of the careers site, do you plan to update it as well?

No, the PL version of the site is not needed and will be removed for the time being.

File: content/careers.en.md (hunk 1 above)
Comment (Stanislaw-bnk) [reply content/careers.en.md]: The links has been removed.

**Review by artur-rs — APPROVED** [review]

**Pull request merged**